addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -55,7 +55,7 @@ def _inverse_l10n_latam_document_number(self):
     def _compute_l10n_latam_sequence(self):
         recs_with_journal_id = self.filtered('journal_id')
         for rec in recs_with_journal_id:
-            rec.l10n_latam_sequence_id = rec._get_document_type_sequence()
+            rec.l10n_latam_sequence_id = rec._get_document_type_sequence()[0]
```

Comment:
Same result but safer:
```suggestion
            rec.l10n_latam_sequence_id = rec._get_document_type_sequence()[:1]
```

Reply:
Thanks, adapted !